<launch>
    <node name="gnss_ins_sim_recorder_node" pkg="gnss_ins_sim" type="recorder_ceshi.py">
        <!-- load default params -->
        <rosparam command="load" file="$(find gnss_ins_sim)/config/recorder_ceshi.yaml" />
        <!-- configuration -->
    </node>
</launch>